<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileManager</class>
 <widget class="QWidget" name="FileManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileManager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="6">
    <widget class="QPushButton" name="cancelBtn">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="okBtn">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="5" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="newDirBtn">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../res/res.qrc">
       <normaloff>:/icon/newDirectory_64.png</normaloff>:/icon/newDirectory_64.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="deleteObjBtn">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../res/res.qrc">
       <normaloff>:/icon/deleteObj_64.png</normaloff>:/icon/deleteObj_64.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
